### DISCUSSION TITLE
eQSL Nicknames in Station Profiles aren't defined

### DISCUSSION THREAD

**Discussion by ClaudeMa** [magicbug/Cloudlog#3240] [Q&A]
hi
after the last update from github today i was unable to import to eqsl.cc
error message is 'eQSL Nicknames in Station Profiles aren't defined`

my eqsl username and password is set in my account with the correct username and password
any idea?
Thanks

**Comment by ClaudeMa** [comment]
Hello solved setting eqslqthnickname directly in database but i have not found in web interface where to set it
TNX claude

**Comment by magicbug** [comment]
I dont think anything has changed, eQSL logins are in your account and qth nicknames are in the station locations area but this codes been the same now for a number of years
